cfy_manager/components/restservice/config/authorization.conf

```diff
@@ -122,6 +129,20 @@ permissions:
 
   cluster_status_get:
     - sys_admin
+  cluster_status_put:
+    - sys_admin
+    - manager_status_reporter
+    - db_status_reporter
+    - broker_status_reporter
+  manager_cluster_status_put:
+    - sys_admin
```

Comment:
I think we can leave just the reporter user

Reply:
Do you mean that it would be `reporter_user` instead? Would it also affect the design of the API?

Reply:
No affect on the API, that access is not needed